data/onPostBuild/llmstxt.ts

```diff
@@ -217,32 +363,135 @@ export const onPostBuild: GatsbyNode['onPostBuild'] = async ({ graphql, reporter
+  });
+
+  // Helper function to serialize pages
+  const serializePages = (
+    pages: Array<{ slug: string; meta: { title: string; meta_description: string }; languages: string[] }>,
+  ) => {
+    for (const page of pages) {
+      const { slug, meta, languages } = page;
+      const { title, meta_description } = meta;
+
+      try {
+        const baseUrl = prefixPath({ url: `/docs/${slug}`, siteUrl, pathPrefix: basePath });
+        const safeTitle = escapeMarkdown(title);
+
+        // Generate base page entry (without language parameter)
+        const baseLink = `[${safeTitle}](${baseUrl})`;
+        const baseLine = `- ${[baseLink, meta_description].join(': ')}`;
+        serializedPages.push(baseLine);
+
+        // Generate language-specific entries if the page has languages
+        // Skip language variants that match the page's primary language (e.g., skip ?lang=go for /getting-started/go)
+        if (languages && languages.length > 0) {
+          // Extract the last part of the slug to check if it matches a language
+          const slugParts = slug.split('/');
+          const slugLastPart = slugParts[slugParts.length - 1];
+
+          // Map slug names to their corresponding language codes
+          const slugToLangMap: Record<string, string> = {
+            dotnet: 'csharp',
+            'objective-c': 'objc',
+          };
+
+          // Get the primary language for this page (either direct match or mapped)
+          const primaryLanguage = slugToLangMap[slugLastPart] || slugLastPart;
+
+          for (const language of languages) {
+            // Skip if the language matches the page's primary language
+            if (language !== primaryLanguage) {
+              const langUrl = `${baseUrl}?lang=${language}`;
+              const langLink = `[${safeTitle} (${getLanguageLabel(language)})](${langUrl})`;
+              const langLine = `- ${[langLink, meta_description].join(': ')}`;
+              serializedPages.push(langLine);
+            }
+          }
         }
+      } catch (err) {
+        reporter.panic(`${REPORTER_PREFIX} Error serializing pages`, err as Error);
       }
-    } catch (err) {
-      reporter.panic(`${REPORTER_PREFIX} Error serializing pages`, err as Error);
+    }
+  };
+
+  for (const categoryKey of sortedCategories) {
+    const category = categoryStructure[categoryKey];
+    serializedPages.push(`## ${category.title}`);
+    serializedPages.push('');
+
+    // Add pages directly under the category (no subcategory)
+    if (category.pages && category.pages.length > 0) {
+      serializePages(category.pages);
+      serializedPages.push(''); // Add blank line after category pages
+    }
+
+    // Sort subcategories alphabetically
+    const sortedSubcategories = Object.keys(category.subcategories).sort();
+
+    for (const subcategoryKey of sortedSubcategories) {
+      const subcategory = category.subcategories[subcategoryKey];
+      serializedPages.push(`### ${subcategory.title}`);
```

Comment:
You need to have a blank line below any heading, so:

~~~suggestion
      serializedPages.push(`### ${subcategory.title}`);
      serializedPages.push('');
~~~